internal/client/client.go

```diff
@@ -0,0 +1,28 @@
+package magento
+
+import (
+	"github.com/go-resty/resty/v2"
+)
+
+// Client struct to hold the Magento API client configuration
+type Client struct {
+	BaseURL     string
+	AccessToken string
+}
+
+// NewClient initializes a new Magento API client configuration
+func NewClient(baseURL, accessToken string) *Client {
+	return &Client{
+		BaseURL:     baseURL,
+		AccessToken: accessToken,
+	}
+}
+
+// CreateRestyClient creates a new resty client with configuration from Client
+func (c *Client) CreateRestyClient() (*resty.Client, error) {
```

Comment:
@ca7alindev Simple Create() don't use customization of methods. This method simple create client.